tools/buildanalyzer/OWNERS

```diff
@@ -1,4 +1,6 @@
 reviewers:
   - bparees
+  - adambkaplan
```

Comment:
@bparees is this maintained any more?

Reply:
oh wow.  i totally forgot this existed.  it's not maintained but it should still work as far as i know.